### DISCUSSION TITLE
Cannot push translations: "Fork creation failed: not found"

### DISCUSSION THREAD

**Discussion by keunes** [WeblateOrg/weblate#6763] [Q&A] [answered]
Hello,
I just set up a new project (AntennaPod website, libre trial), added a component and made some translations. However, now I want to push the translations back to the source repo, and now I get an error:

![image](https://user-images.githubusercontent.com/11229646/140625467-607034da-f904-4234-97ac-f5960353c8a4.png)

In the version control settings I have set the `Repository push URL` the same as the `Source code repository`. I just added Weblate as a collaborator for my repo (didn't change anything; still getting the error). The docs say:
> Once the `weblate` user is added, you can configure [Source code repository](https://docs.weblate.org/en/latest/admin/projects.html#component-repo) and [Repository push URL](https://docs.weblate.org/en/latest/admin/projects.html#component-push) using the SSH protocol (for example `git@github.com:WeblateOrg/weblate.git`).

However, the linked page about the repository push URL doesn't specify what the URL should be like, or where I could find the URL.

https://docs.weblate.org/en/latest/admin/continuous.html#push-changes on the other hand, suggests that I can leave empty the Repository push URL when using GitHub.

I bet it's some wrong setting, but I'm at a loss what it might be.

**Comment by nijel** [comment, marked as answer]
It seems to be fixed now, can you please share what did help you?

**Reply by keunes** [reply]
Hello!
First I used this fork as the Source code repository: https://github.com/keunes/antennapod.github.io
Then I figured since a requirement for a free Libre project a requirement is that translations cannot be sent to a fork, that might be a problem. So for testing purposes I created a not-fork copy of the website repo, which I then set as the Source code repository: https://github.com/keunes/AP-website-copy

I am not sure, but I think that 'fixed' it - could that be the case? (If so, it might be helpful for other newbies like me to have a bit more explicit error message, noting that it fails because Libre projects are not allowed to submit to forks.)

**Reply by nijel** [reply]
The reason for not allowing forks is to avoid submissions from outside contributors - we want to avoid dealing with a random group of people deciding to translate a project to a single language. The libre hosting is for the cases when project decides for that.

In this case, something went wrong, the error is definitely not intentional. Weblate tried to fork `keunes/antennapodhub.io` instead of `keunes/antennapod.github.io`. That should be fixed in 70d557c04f451151d2ca6670ed467c1adcf7a1b8.

**Reply by keunes** [reply]
> we want to avoid dealing with a random group of people deciding to translate a project to a single language

Yup, totally get the rationale :)

> In this case, something went wrong, the error is definitely not intentional ... That should be fixed in 70d557c.

Super, thank you!
